sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -124,6 +124,40 @@ def __repr__(self):
             .format(self.id, repr(self.entities), repr(self.warnings), repr(self.statistics), self.is_error)[:1024]
 
 
+class RecognizePiiEntitiesResult(DictMixin):
+    """RecognizePiiEntitiesResult is a result object which contains
+    the recognized Personally Identifiable Information (PII) entities
+    from a particular document.
+    
+    :ivar str id: Unique, non-empty document identifier that matches the
+        document id that was passed in with the request. If not specified
+        in the request, an id is assigned for the document.
+    :ivar entities: Recognized PII entities in the document.
+    :vartype entities:
+        list[~azure.ai.textanalytics.PiiEntity]
+    :ivar warnings: Warnings encountered while processing document. Results will still be returned
+     if there are warnings, but they may not be fully accurate.
```

Comment:
nit: indent level

Reply:
ok. all of my previous `warnings` docstrings just used the one space, so I also changed those for the sake of consistency